HolyShift.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
Please follow the folder structure convention

<RepoRoot>/
|  *.sln
|  <Proj1>/
|  |  *.cs
|  |  <Proj1>.csproj
|  <Proj2>/
|  |  *.cs
|  |  <Proj2>.csproj
...